### DISCUSSION TITLE
HLS to file or mabye sftp/ftp output destination

### DISCUSSION THREAD

**Discussion by FilipStadler** [datarhei/restreamer#436] [Ideas]
I am really missing a record a stream option.

Maybe even one I can mount using sftp but even just saving to file in DATA location would also be very usable.

Currently only http/https is a option... 

its posible to use any http/https server as hls playout location if the player handle the location.

nginx can record a flv file that can be translated later sometimes better to do later maybe even by a remote platform with more gpu power.

Sometimes when I copy paste url - the procotol is assign already protocol://  so I always need to edit the url and remove the protocol where I think if https,http,srt,rtmp is assign allready when user past in a url with "protocol://" the restreamer should know its safe to fix the url - it will just make it easy to copy/fast this not realated to the HLS but other places like rtmp and srt.

**Comment by svenerbeck** [comment]
Hello Filip
Good to hear from you, and thanks for the input. Video also includes recording — not just live. You are right. It is a logical requirement for different types of projects. A module for recording will probably come in 2023.

Greetings
Sven

PS: Take a look at your Discord 🎁
